### DISCUSSION TITLE
SQLx and Glommio

### DISCUSSION THREAD

**Discussion by shivshankardayal** [ntex-rs/ntex#377] [Q&A]
I know that `SQLx` does not support `Glommio`. But, `ntex` gives us that option. I know that `SQLx` does not support `Glommio`. How can I make use of `GLommio` with `ntex` and `SQLx`?

**Comment by fafhrd91** [comment]
sqlx has to support glommio, ntex cant magically
make it compatible.

**Reply by shivshankardayal** [reply]
I feel like an idiot asking that question. Of course, you are right. BTW, you are no longer part of ntex-rs org?

**Reply by fafhrd91** [reply]
i am member of org, just was private
